pages/docs/tutorials/native/interop-with-c.md

```diff
@@ -0,0 +1,140 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Interop with C Libraries"
+description: "A look at how to interop with C libraries"
+authors: Hadi Hariri 
+date: 2018-01-09
+showAuthorInfo: false
+---
+
+
+When writing native applications, oftentimes we need to access certain functionality that is not included in the Kotlin standard library, 
+such as making HTTP calls, reading and writing from disk, etc.
+
+Kotlin/Native provides us with the ability to consume standard C libraries, opening up an entire ecosystem of functionality that exists 
+for pretty much anything we could need. 
```

Comment:
Maybe add something like
"Also note, that platform libraries allows one to use native for the platform functionality without need of creating any interoperability definitions."

See smth like https://github.com/JetBrains/kotlin-native/blob/master/PLATFORM_LIBS.md and https://github.com/JetBrains/kotlin-native/blob/master/samples/csvparser/src/main/kotlin/CsvParser.kt as an example.